binance/client.py

```diff
@@ -5508,6 +5558,14 @@ def futures_coin_get_position_mode(self, **params):
 
         """
         return self._request_futures_coin_api("get", "positionSide/dual", True, data=params)
+    
+    def fetch_all_coins_information(self, **params):
```

Comment:
Copy of this function `get_all_coins_info`

Reply:
hmph. I was going to bring this as separate PR. Sorry about confusion. Will do.